LICENSE.txt

```diff
@@ -0,0 +1,165 @@
+                   GNU LESSER GENERAL PUBLIC LICENSE
+                       Version 3, 29 June 2007
```

Comment:
I'm a little wary of changing the licensing of old code. I see that Doug Auclair put "opyright: (c) 2002, LGPL" in the original commit so I wouldn't change it to the 2007 version of LGPL, which could be different.  It looks like LGPL-2.1 was in effect in 2002. Update to that instead?

Reply:
My license-foo is weak, I thought that this LGPL was the same. I'll change it.